packages/injected/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/injected-wallets",
-  "version": "2.3.0",
+  "version": "2.3.1-alpha.1",
```

Comment:
Are there changes in the injected package that require a version bump? I see the bump for the common package, but there are no changes I can see in the common package.